src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/tests/latest/test_monitor_log_profile.py

```diff
@@ -0,0 +1,25 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer, StorageAccountPreparer
+
+
+class TestActionGroupScenarios(ScenarioTest):
+    @ResourceGroupPreparer(location='southcentralus')
+    @StorageAccountPreparer(location='southcentralus')
+    def test_monitor_create_log_profile(self, resource_group, storage_account):
+        name = self.create_random_name('clitest', 20)
+        storage = self.cmd('storage account show -n {} -g {} --query id -otsv'.format(storage_account, resource_group))
```

Comment:
Recommend using the kwargs style for readability.

Reply:
Should me an example?

Reply:
From network:

```Python
self.kwargs.update({
            'asg': 'asg1'
        })

        count1 = len(self.cmd('network asg list').get_output_in_json())
        self.cmd('network asg create -g {rg} -n {asg}',
                 checks=self.check('tags', None))
        self.cmd('network asg update -g {rg} -n {asg} --tags foo=bar',
                 checks=self.check('tags.foo', 'bar'))
        count2 = len(self.cmd('network asg list').get_output_in_json())
        self.assertTrue(count2 == count1 + 1)
        self.cmd('network asg show -g {rg} -n {asg}', checks=[
            self.check('name', '{asg}'),
            self.check('resourceGroup', '{rg}'),
            self.check('tags.foo', 'bar')
```

You'll need 95% fewer .format statements, making it easier to read the test and just know what's happening :)